### PR TITLE
CMakeList Fixes

### PR DESCRIPTION
**Pull request by kiritigowda**
* ROCm Path Fix
* Installation Path Set
* Message print

### CODE REVIEW

**Comment by kiritigowda** [comment]
@LakshmiKumar23 can you install this version and check if MIVisionX works right. Logs below

https://travis-ci.org/github/GPUOpen-ProfessionalCompute-Libraries/rpp

**Comment by LakshmiKumar23** [comment]
@kiritigowda This PR builds. Tested MIVisionX with this version. That also builds.

**Comment by rrawther** [comment]
@LakshmiKumar23 : Can you build both rpp and MIVisionX, run test .

File: CMakeLists.txt [hunk 1]
```diff
@@ -304,9 +315,5 @@ target_include_directories(${PROJECT_NAME}
 add_subdirectory(addkernels)
 
 # Installation
-install (TARGETS amd_rpp
-         LIBRARY DESTINATION /opt/rocm/rpp/lib
-)
-install (DIRECTORY include
-         DESTINATION /opt/rocm/rpp/
-)
+install (TARGETS amd_rpp LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib)
```
Comment (kiritigowda) [inline CMakeLists.txt:318]: @LakshmiKumar23 can you check if this works

**Comment by kiritigowda** [comment]
@LakshmiKumar23 is this good to merge?

File: CMakeLists.txt (hunk 1 above)
Comment (LakshmiKumar23) [reply CMakeLists.txt]: Yes rpp gets install at this path

**Pull request merged**